phone/src/os/call/components/CallContactContainer.tsx

```diff
@@ -21,13 +21,15 @@ const CallContactContainer = () => {
             ? t('CALLS.MESSAGES.OUTGOING').toUpperCase()
             : t('CALLS.MESSAGES.INCOMING').toUpperCase()}
         </Typography>
-        <Typography variant="h4">{getDisplayOrNumber()}</Typography>
+        <Typography variant="h4">{call.label || getDisplayOrNumber()}</Typography>
       </Box>
       <Avatar
         sx={{ ml: 1, height: 80, width: 80 }}
-        alt={getDisplayOrNumber()}
+        alt={call.label || getDisplayOrNumber()}
         src={
-          call.isTransmitter
+          call.label
```

Comment:
Maybe this can be a function instead of a ternary within a ternary just for clarity

Reply:
true, will fix when i rebase.